lib/documentation.dart

```diff
@@ -106,7 +76,48 @@ ${info['libraryName'] == null ? "" : "**Library:** $apiLink" }\n\n
     _context.dartSource.substring(offset);
   }
 
-  String _dartApiLink({String libraryName, String enclosingClassName, String memberName}){
+  Future<_DocResult> _getHtmlTextFor(DocumentResponse result) {
+    Map info = result.info;
```

Comment:
This might be better served at some point by wrapping our own class around the Map, which can provide semantic access to the map's contents.